### PR TITLE
Adapt cloud to v4.2.1

### PR DESCRIPTION
**Pull request by SitoRBJ**
Hello team,

This issue adapts cloud code to use Wazuh v4.2.0

Waiting merge for official release.

Regards

### CODE REVIEW

File: wazuh/config/01-wazuh.sh [hunk 1]
```diff
@@ -35,38 +35,54 @@ exec_cmd_stdout() {
 ##############################################################################
 # Check_update
 # This function considers the following cases:
-# - If /var/ossec/etc/ossec-init.conf does not exist -> Action Nothing. There is no data in the EBS. First time deploying Wazuh
-# - If /var/ossec/etc/VERSION does not exist -> Action: Update. The previous version was prior to 3.11.5.
-# - If both files exist: different Wazuh version -> Action: Update. The previous version is older than the current one.
-# - If both files exist: the same Wazuh version -> Acton: Nothing. Same Wazuh version.
+# - If /var/ossec/etc/VERSION does not exist -> Action: Update. Action Nothing. There is no data in the EBS. First time deploying Wazuh
```
Comment (jhaos) [inline wazuh/config/01-wazuh.sh:38]: Why there is 2 actions instead one of them?

Comment (SitoRBJ) [reply wazuh/config/01-wazuh.sh]: solved

File: wazuh/config/01-wazuh.sh [hunk 2]
```diff
@@ -35,38 +35,54 @@ exec_cmd_stdout() {
...
-      previous_version=$(cat /var/ossec/etc/VERSION | grep -i version | cut -d'"' -f2)
-      echo "Previous version: $previous_version"
-      current_version=$(cat ${WAZUH_INSTALL_PATH}/data_tmp/permanent/var/ossec/etc/ossec-init.conf | grep -i version | cut -d'"' -f2)
-      echo "Current version: $current_version"
-      if [ $previous_version == $current_version ]
+      echo "Same Wazuh version in the EBS and image"
+      return 0
+    else
+      echo "Different Wazuh version: Update"
+      if [ $previous_version == "v4.1.5" ]
```
[the remark this answers is not in the document]
Comment (SitoRBJ) [reply wazuh/config/01-wazuh.sh:56]: This action is only necessary with this specific version.

**Review by jhaos — APPROVED** [review]
LGTM!!

**Pull request merged**